programmer-humour/script.js

```diff
@@ -0,0 +1,32 @@
+
+function getImageFetch(){
+  return fetch('https://xkcd.now.sh/?comic=latest').then((response) => {
+    if(!response.ok){
+        return "Error";
```

Comment:
Instead of just returning the text "Error", we could instead throw an exception. This is a large topic to cover here, so reach out on Slack if you have any questions on what this means!